### PR TITLE
[user-agent] User agent override for forbes.com. Fixes JB#51732

### PR DESCRIPTION
**Pull request by mkenttala**

### CODE REVIEW

**Comment by rainemak** [comment]
Re-direct to next branch and rebase.

File: data/ua-update.json [hunk 1]
```diff
@@ -65,5 +65,6 @@
   "google.at": "Linux#Maemo",
   "twitter.com": "Linux#Android 8.1.0",
   "apkpure.com": "Linux#Android 8.1.0",
-  "zingmp3.vn": "Linux#Android 8.1.0"
+  "zingmp3.vn": "Linux#Android 8.1.0",
+  "forbes.com": "Linux#Android 8.1.0"
```
Comment (rainemak) [inline data/ua-update.json:69]: Btw, would dropping Linux be enough?

Like:
```
"forbes.com": "Linux# "
```

Space after the hash

Comment (mkenttala) [reply data/ua-update.json:69]: Nope, needs Android. Works without that "8.1.0" though.

Comment (rainemak) [reply data/ua-update.json:69]: Others have that 8.1.0 => so we can be it with this as well.

**Review by rainemak — APPROVED** [review]
Looks good.

**Pull request merged**